Doc/library/threading.rst

```diff
@@ -654,7 +672,7 @@ Reentrant locks also support the :ref:`context management protocol <with-locks>`
 
       Only call this method when the calling thread owns the lock. A
       :exc:`RuntimeError` is raised if this method is called when the lock is
-      unlocked.
+      un-acquired.
```

Comment:
```suggestion
      not acquired.
```

Un-acquired would be interpreted as verb here, where this is in fact a noun (a state)